### DISCUSSION TITLE
press and hold to select an option in a dropdown menu

### DISCUSSION THREAD

**Discussion by vioco** [ppy/osu#22065] [Ideas]
osu!stable has this feature where you can switch between collections or leaderboard by pressing the dropdown button,
![screenshot219](https://user-images.githubusercontent.com/47917132/211173022-93321477-9d56-486d-bef0-258db0f5add9.png)
holding down 
![screenshot218](https://user-images.githubusercontent.com/47917132/211173030-0e504e47-e1dd-4869-97bc-ca51f35ec7ec.png)
and releasing on the selected option.
![screenshot220](https://user-images.githubusercontent.com/47917132/211173037-dded1160-4ed4-42d3-9475-d6c04b97eb33.png)
I think this would be a great feature for the dropdowns in osu lazer, for example for when selecting collections
![osu_2023-01-07_23-37-16](https://user-images.githubusercontent.com/47917132/211173069-a8ee13a9-17fb-4368-8472-ccbe7c0e8b7f.jpg)
or adding maps to collections.
![osu_2023-01-07_23-38-33](https://user-images.githubusercontent.com/47917132/211173090-ddcf0251-b122-42c5-846d-1fbd7574716c.jpg)
I found this to be a great feature for navigating the menus with a tablet on osu!stable because pointing and clicking with a tablet is often quite difficult depending on the grip. This effectively reduces 2 point and click events (or more) to just one.

**Comment by yuuiko** [comment]
drag to select
